snorkel/map/core.py

```diff
@@ -28,19 +33,90 @@ def get_parameters(
     return params[0]
 
 
+def is_hashable(obj: Any) -> bool:
+    """Test if object is hashable via duck typing.
+
+    NB: not using `collections.Hashable` as some objects
```

Comment:
NB?

Reply:
By usage, I assume this just means "Note:", but the latter seems clearer?

Reply:
(Whatever we decide, update all instances of NB)

Reply:
This is going to be so pretentious. https://en.wikipedia.org/wiki/Nota_bene. It means "Important note". Pretty common in code comments.

Reply:
Hahahaha I literally read Braden's first comment, and was like- I can't wait for Henry's response it's going to be great ("Well, on the Yale Daily News, we used it for...")

Reply:
However I also have seen it a lot fwiw, although I also default to 'Note:'